src/import-service/importProductsFile/index.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+exports.handler = async (event) => {
+    const { getSignedUrl } = require("@aws-sdk/s3-request-presigner");
+    const { PutObjectCommand, S3Client } = require("@aws-sdk/client-s3");
+    
+    const { queryStringParameters: { name } } = event;
+    const client = new S3Client({ region: "eu-central-1" });
+    const getObjectParams = {Bucket: 'cloudx-aws-practitioner-js', Key: `uploaded/${name}`};
+    const command = new PutObjectCommand(getObjectParams);
+    const url = await getSignedUrl(client, command, { expiresIn: 3600 });
```

Comment:
I'd suggest you use **try...catch** statement when you're using **await**. Or you can switch to Promises but then you'd have to use "then-catch-finally" combo. In the catch-block of try...catch it's a common practice to do something with the error that can be thrown while you're trying to do some async stuff.